docs/static/platforms.html

```diff
@@ -0,0 +1,45 @@
+<head>
+<style>
+
+.mdl-grid {
+    align-items: center;
+    justify-content: flex-start;
+    text-align: center;
+    padding: 20px 0 0 0;
+}
+
+.running-item  {
+  max-width: 300px;
+}
+
+.running-item img {
+  height: 40px;
+}
+
+.running-item p {
+    padding: 20px;
+}
+
+</style>
+</head>
+<div class="content">
+	<div class = "mdl-grid running">
+		<div class="running-item">
+			<a href="https://github.com/gidler/autogluon-tutorials/blob/main/welcome-notebook.ipynb">
```

Comment:
Do we expect `ipynb` files to live in the other repo? What's the long-term strategy for these?

Reply:
For sagemaker studio lab, the notebooks/ipynb are more static. (e.g. in D2L, we have a [separate git repo](https://github.com/d2l-ai/d2l-pytorch-sagemaker-studio-lab) for hosting studio lab notebooks.)

Reply:
In a previous conversation Nick suggested I make this new repo to host the notebooks. In the future we may (should) find a better place for them to live long term, but my understanding is creating a new awslab project is prohibitively difficult. Happy to discuss more